<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UR10 GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>UR10 Config</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>PORT</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="btnConnect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="tbIP"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="tbPort"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>UR10 Movement</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="5" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>RZ</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="tbRy"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="tbRz"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="tbRx"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="tbTime"/>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Elbow</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>RY</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QPushButton" name="btnMove">
              <property name="text">
               <string>Move</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Time</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Z</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="tbX">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>RX</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="tbZ"/>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="tbBase"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Shoulder</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="tbY">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Base</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="tbShoulder"/>
            </item>
            <item row="13" column="1">
             <widget class="QLineEdit" name="tbWrist1"/>
            </item>
            <item row="11" column="1">
             <widget class="QLineEdit" name="tbElbow"/>
            </item>
            <item row="17" column="1">
             <widget class="QLineEdit" name="tbWrist3"/>
            </item>
            <item row="14" column="1">
             <widget class="QLineEdit" name="tbWrist2"/>
            </item>
            <item row="13" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Wrist 1</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Wrist 2</string>
              </property>
             </widget>
            </item>
            <item row="17" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Wrist 3</string>
              </property>
             </widget>
            </item>
            <item row="23" column="0" colspan="2">
             <widget class="QPushButton" name="btnMoveJoints">
              <property name="text">
               <string>Move Joints</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>label_3</zorder>
           <zorder>tbX</zorder>
           <zorder>label_4</zorder>
           <zorder>tbY</zorder>
           <zorder>label_5</zorder>
           <zorder>tbZ</zorder>
           <zorder>label_6</zorder>
           <zorder>tbRx</zorder>
           <zorder>label_7</zorder>
           <zorder>tbRy</zorder>
           <zorder>label_8</zorder>
           <zorder>tbRz</zorder>
           <zorder>btnMove</zorder>
           <zorder>tbTime</zorder>
           <zorder>label_9</zorder>
           <zorder>label_12</zorder>
           <zorder>label_10</zorder>
           <zorder>tbBase</zorder>
           <zorder>label_11</zorder>
           <zorder>tbWrist1</zorder>
           <zorder>tbElbow</zorder>
           <zorder>tbShoulder</zorder>
           <zorder>tbWrist2</zorder>
           <zorder>tbWrist3</zorder>
           <zorder>btnMoveJoints</zorder>
           <zorder>label_13</zorder>
           <zorder>label_14</zorder>
           <zorder>label_15</zorder>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>UR10 Status</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="tbStatus"/>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="btnAddJoint">
              <property name="text">
               <string>Add Joints</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QListWidget" name="lbPoses"/>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="btnAddPosition">
              <property name="text">
               <string>Add Position</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Positions/Joints</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="tbName"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Name of position/joint</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QPushButton" name="btnSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnUpdatePosJoints">
              <property name="text">
               <string>Update positions and joints</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
